packages/chat-stateful-client/src/EventSubscriber.ts

```diff
@@ -44,6 +44,13 @@ export class EventSubscriber {
     });
   };
 
+  // convert event type to chatMessage type, only possible type is 'html' and 'text' in chat event
+  private convertEventType = (type: string): ChatMessageType => {
+    const lowerCaseType = type.toLowerCase();
+    if (lowerCaseType === 'richtext/html' || lowerCaseType === 'html') return 'html';
```

Comment:
I am curious these types are used across all of chat. Do we not have shared code that can always do this conversion?